client/beefy/src/worker.rs

```diff
@@ -254,6 +268,8 @@ where
 	R: ProvideRuntimeApi<B>,
 	R::Api: BeefyApi<B> + MmrApi<B, MmrRootHash, NumberFor<B>>,
 	N: NetworkEventStream + NetworkRequest + SyncOracle + Send + Sync + Clone + 'static,
+	u64: From<NumberFor<B>>,
+	<<B as BlockT>::Header as HeaderT>::Number: From<u64>,
```

Comment:
where are we converting `NumberFor<B>` -> `u64`?

Reply:
In this method here https://github.com/dharjeezy/substrate/blob/dharjeezy/bounds-on-enqueued-votes/client/beefy/src/worker.rs#L562